src/charts/line/lineDataOnlyNonzeroPositiveValues.json

```diff
@@ -0,0 +1,42 @@
+{
+	"dataByTopic": [
```

Comment:
We are moving out of this kind of data shape and into a more flat structure, check out the other .json files in this folder to see it in practice.

It would be great if you could use that shape.